### ISSUE TITLE
Fix test-arm-reference-delegation

### ISSUE THREAD

**Issue by guangy10** [pytorch/executorch#1750]
```
+ bash backends/arm/test/run_tosa_reference.sh
/opt/conda/envs/py_3.10/lib/python3.10/site-packages/transformers/utils/generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(

Processing:::simple_clone
Debug output path for intermediates: /tmp/arm_tosa_iie9okrj
/tmp/arm_tosa_iie9okrj/simple_clone/torch/ /tmp/arm_tosa_iie9okrj/simple_clone/tosa/
/bin/sh: 1: tosa_reference_model: not found
Traceback (most recent call last):
  File "/pytorch/executorch/backends/arm/test/arm_tosa_reference.py", line 296, in <module>
    tosa_run_test(op, profile=TosaProfile.MI)
  File "/pytorch/executorch/backends/arm/test/arm_tosa_reference.py", line 230, in tosa_run_test
    subprocess.run([cmd_ref_model], shell=True, check=True)
  File "/opt/conda/envs/py_3.10/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['tosa_reference_model --test_desc /tmp/arm_tosa_iie9okrj/simple_clone/tosa//desc.json']' returned non-zero exit status 127.
```
See full stack trace in: https://github.com/pytorch/executorch/actions/runs/7705685481/job/21000020334

**Comment by guangy10** [comment]
CC: @Jerry-Ge

**Comment by Jerry-Ge** [comment]
Thanks @guangy10 , it's probably due to the missing `-b` here: https://github.com/pytorch/executorch/blob/7b5419a706db7e1949f4152a9e184753d35a52bc/examples/arm/setup.sh#L170

Should be : `git clone https://review.mlplatform.org/tosa/reference_model -b v0.80`

**Comment by guangy10** [comment]
Put up a quick fix in #1761

**Comment by robell** [comment]
working after #2143 merged.
